### PR TITLE
add delegation introduction and suggestions

### PR DESCRIPTION
**Pull request by christian-posta**

### CODE REVIEW

**Comment by christian-posta** [comment]
cc @scranton

File: docs/content/introduction/decentralized_routing.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Decentralized ownership"
+weight: 31
+---
+
+## Decentralized API ownership
+Gloo’s developer-centric workflows can benefit from two specific parts of the Gloo configuration API. The first is service teams can completely own the {{< protobuf name="gateway.solo.io.VirtualService" display="VirtualService">}} configuration in a “build it you run it” manner. The second is a [delegation model]({{% versioned_link_path fromRoot="/gloo_routing/virtual_services/delegation/" %}}) that allows service teams to own certain aspects of the configuration while allowing a centralized API team to own some of the [security and operation aspects]({{% versioned_link_path fromRoot="/security/" %}}). Let’s take a look at the second approach as it’s most applicable to our users and the workflows they wish to adopt.
```
Comment (marcogschmidt) [inline docs/content/introduction/decentralized_routing.md]:
```suggestion
Gloo’s developer-centric workflows can benefit from two specific parts of the Gloo configuration API. The first is service teams can completely own the {{< protobuf name="gateway.solo.io.VirtualService" display="VirtualService">}} configuration in a “you build it you run it” manner. The second is a [delegation model]({{% versioned_link_path fromRoot="/gloo_routing/virtual_services/delegation/" %}}) that allows service teams to own certain aspects of the configuration while allowing a centralized API team to own some of the [security and operation aspects]({{% versioned_link_path fromRoot="/security/" %}}). Let’s take a look at the second approach as it’s most applicable to our users and the workflows they wish to adopt.
```

Added a `you` here: “you build it you run it”

**Pull request merged**